PYME/Acquire/Hardware/pco/pco_sdk_cam.py

```diff
@@ -433,7 +489,9 @@ def _init_buffers(self):
 
     @property
     def _timeout(self):
-        return int(max(2*100*self.GetCycleTime(), 100))
+        # set the time longer for all hardware check
```

Comment:
This timeout should be specific to the camera (other hardware and hardware checks are completely unimportant here). It's used in the tight loop which just serves the camera. Setting this too long is likely to be undesirable (makes it more likely that the camera buffer is not kept full).